Centaurus.Domain/WebSockets/Alpha/Info/Commands/SubscribeCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.Domain
+{
+    [Command("Subscribe")]
+    public class SubscribeCommand: BaseCommand
+    {
+        public List<string> Subscriptions { get; set; }
```

Comment:
Could you please provide an example of a call in JSON format? How do you encode price history for BTC market and 15m period?